lhotse/dataset/source_separation.py

```diff
@@ -5,9 +5,7 @@
 from torch.utils.data import Dataset
 
 from lhotse.cut import CutSet, Cut, AnyCut
-from lhotse.utils import Pathlike
-
-EPS = 1e-8
```

Comment:
It's not very clear on reading  that EPS refers to epsilon.  I wonder if EPSILON or epsilon might be clearer.